.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Can you remove these files in `/.idea`, maybe add `/.idea` to .gitignore?

Reply:
Ah god didn't even notice. Yeah.